torch/_inductor/compile_fx.py

```diff
@@ -576,6 +582,39 @@ def compile_fx_inner(
         )
 
 
+_PRECOMPILES: Dict[str, List[Any]] = {}
```

Comment:
Not thread safe.  I'd think we should be able to eliminate the name and store this on the `torch.compile` object (just need to thread a pointer to that object down into this function).